app/controllers/publishers_controller.rb

```diff
@@ -554,4 +591,12 @@ def prompt_for_two_factor_setup
   def update_sendgrid(publisher:, prior_email: nil)
     RegisterPublisherWithSendGridJob.perform_later(publisher.id, prior_email)
   end
+
+  def no_channels_banner_data
+    return {
+      "channels": [{"id" => "00000000-0000-0000-0000-000000000000", "name" => " ", "type" => "SiteChannelDetails"}],
```

Comment:
Preferably we use symbols as opposed to strings. Aside from the slight performance improvement, they're immutable.

Reply:
Done ✔️